source/whats-new.txt

```diff
@@ -12,12 +12,28 @@ What's New
 
 Learn what's new in:
 
+* :ref:`Version 4.5 <version-4.5>`
 * :ref:`Version 4.4 <version-4.4>`
 * :ref:`Version 4.3 <version-4.3>`
 * :ref:`Version 4.2 <version-4.2>`
 * :ref:`Version 4.1 <version-4.1>`
 * :ref:`Version 4.0 <version-4.0>`
 
+.. _version-4.5:
+
+What's New in 4.5
+-----------------
+
+New features of the 4.5 Java driver release include:
+
+- Added support for specifying custom service names of SRV resource
+  records. See the ``srvServiceName`` setting in the :ref:`Connection
+  Options <connection-options>` guide for more information.
+- Added support to customize UUID representation to in a registry using
+  the ``CodecRegistries.withUuidRepresentation()`` method.
+- Added support for overriding listener settings in ``ClusterSettings``,
+  ``ServerSettings`` and ``ConnectionPoolSettings`` builders.
```

Comment:
I think we should also mention the following changes:

- https://jira.mongodb.org/browse/JAVA-4449. As a result of this change, all events received by `ClusterListener`, `ServerListener`, `ServerMonitorListener` are totally ordered provided that the listeners are not shared by different `MongoClient`s.
- https://jira.mongodb.org/browse/JAVA-4440
- https://jira.mongodb.org/browse/JAVA-4452. Improved the performance characteristics of 4.5 when compared with 4.4 and 4.3. Now they are back to be on par with 4.2.